test/ng/compileSpec.js

```diff
@@ -980,6 +980,29 @@ describe('$compile', function() {
         }));
 
 
+        it('should not set merged attributes twice in $attrs', function() {
+          var attrs;
+
+          module(function() {
+            directive('logAttrs', function() {
+              return {
+                link: function($scope, $element, $attrs) {
+                  attrs = $attrs;
+                }
+              }
+            })
+          });
+
+          inject(function($compile, $rootScope) {
```

Comment:
Aren't `$compile` and `$rootScope` already available inside the top-level `describe's scope ?
Why is everyone injecting it ? :stuck_out_tongue: